### ISSUE TITLE
[Api] Catalog State Down

### ISSUE THREAD

**Issue by dwlodarczyk13** [cyborgbackup/cyborgbackup#76]
Hello,

I've managed to make the project up and running, but I feel like I'm stuck on last step maybe (?).

I cannot make the `Catalog` go green no matter what I do:

![image](https://github.com/cyborgbackup/cyborgbackup/assets/85172844/e417a322-c2cc-41f2-9e25-b35b9df94919)

The `catalog` job has no log entry at all (the log window and file are empty)

![image](https://github.com/cyborgbackup/cyborgbackup/assets/85172844/40712d08-ff0a-45a5-bc37-c3b631e98b53)

Catalog in GUI is empty

![image](https://github.com/cyborgbackup/cyborgbackup/assets/85172844/a2b49df7-beb0-485a-a23e-75caedf3aeb3)

I didn't managed to make the `Backup Job` yet run succesfully, as I need to get some externally reachable `ftp server` yet, but can it be that the catalog keeps being empty because the `client` fails calling the `repo` (as mentioned, the target `ftp` is only available locally for containers now`), so `celery` sees it, client ofc not.

**Comment by dwlodarczyk13** [comment]
> can it be that the catalog keeps being empty because the client fails calling the repo

Yes

![image](https://github.com/cyborgbackup/cyborgbackup/assets/85172844/ce4daa3c-e247-4a78-ac65-7b2187eb5aff)

**Comment by Ubikfr** [comment]
Same here.
The port `27017 ` is mongodb’s and the API container shouldn’t look for it on himself (localhost) but rather on mongodb’s container.
I looked into the code and at some point there is a test if `MONGODB_HOST` is defined else fallback to 127.0.0.1.
There is not`MONGODB_HOST` in the documented .env file.

**Comment by dwlodarczyk13** [comment]
> Same here. The port `27017` is mongodb’s and the API container shouldn’t look for it on himself (localhost) but rather on mongodb’s container. I looked into the code and at some point there is a test if `MONGODB_HOST` is defined else fallback to 127.0.0.1. There is not`MONGODB_HOST` in the documented .env file.

Maybe that's the problem. Maybe catalog runs on mongo.

For me mongo did not even started at all with `docker-compose`, had to start it manually each time. The fact that everything works without mongo up, would sugest that mongo is maybe indeed used by the `catalog`.

Starting mongo **did not** solved the problem on its own.

**Comment by Ubikfr** [comment]
> Starting mongo **did not** solved the problem on its own.

No, the API container needs to be able to connect to it and he can’t …
I did add `MONGODB_HOST=mongodb` in the .env file, restart the API container, verify that the API container can resolve and ping `mongodb` but he still tries to connect to localhost : the ENV variable isn’t parsed
